internal/cli/atlas/networking/peering/create/aws_test.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build unit
+
+package create
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongocli/internal/mocks"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func TestAwsOpts_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockPeeringConnectionCreator(ctrl)
+	defer ctrl.Finish()
+
+	opts := &AWSOpts{
+		store:  mockStore,
+		region: "TEST",
+	}
+	t.Run("container exists", func(t *testing.T) {
+		containers := []mongodbatlas.Container{
+			{
+				RegionName: opts.region,
+			},
+		}
+		mockStore.
+			EXPECT().
+			AWSContainers(opts.ProjectID).
+			Return(containers, nil).
+			Times(1)
+
+		request := opts.newPeer("")
+		mockStore.
+			EXPECT().
+			CreatePeeringConnection(opts.ProjectID, request).
+			Return(&mongodbatlas.Peer{}, nil).
+			Times(1)
+		if err := opts.Run(); err != nil {
+			t.Fatalf("Run() unexpected error: %v", err)
+		}
+	})
+	t.Run("container does not exist", func(t *testing.T) {
+		mockStore.
+			EXPECT().
+			AWSContainers(opts.ProjectID).
+			Return(nil, nil).
+			Times(1)
+		containerRequest := opts.newContainer()
+		mockStore.
+			EXPECT().
+			CreateContainer(opts.ProjectID, containerRequest).
+			Return(&mongodbatlas.Container{ID: "ID"}, nil).
+			Times(1)
+
+		request := opts.newPeer("ID")
+		mockStore.
+			EXPECT().
+			CreatePeeringConnection(opts.ProjectID, request).
+			Return(&mongodbatlas.Peer{}, nil).
+			Times(1)
+		if err := opts.Run(); err != nil {
+			t.Fatalf("Run() unexpected error: %v", err)
+		}
+	})
+}
+
+func TestNormalizeAtlasRegion(t *testing.T) {
```

Comment:
let's use a [table based](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests) test here
if you try to generate the test with IDEA, IDEA already gives you the skeleton for this type of test